unlock-app/src/components/interface/keychain/AddToPhoneWallet.tsx

```diff
@@ -0,0 +1,174 @@
+import { Modal } from '@unlock-protocol/ui'
+import QRCode from 'qrcode.react'
+import { DiAndroid as AndroidIcon, DiApple as AppleIcon } from 'react-icons/di'
+import { createWalletPass, Platform } from '../../../services/ethpass'
+import { useAuth } from '~/contexts/AuthenticationContext'
+import { ToastHelper } from '~/components/helpers/toast.helper'
+import { WalletService } from '@unlock-protocol/unlock-js'
+
+// https://stackoverflow.com/questions/9038625/detect-if-device-is-ios
+function iOS() {
+  return (
+    [
+      'iPad Simulator',
+      'iPhone Simulator',
+      'iPod Simulator',
+      'iPad',
+      'iPhone',
+      'iPod',
+    ].includes(navigator.platform) ||
+    // iPad on iOS 13 detection
+    (navigator.userAgent.includes('Mac') && 'ontouchend' in document)
+  )
+}
```

Comment:
Not 100% sure but I think we already do this check somewhere in the code using userAgent check. 

There is a package to do this for all platforms and mobile vs desktop browser so we can replace it with that instead of repeating this code.

Reply:
Added!